packages/ffe-lists/less/regular-lists.less

```diff
@@ -138,6 +141,14 @@
         }
     }
 
+    > li.ffe-check-list__item--cross {
+        &::before {
+            background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 200 200'%3E%3Cpath fill='@{ffe-red-fill}' d='M18.77,0c-3.562,0-7.125,1.347-9.834,4.044l-4.429,4.439c-5.381,5.394-5.381,14.25,0,19.68l71.19,71.83-71.66,71.83c-5.381,5.394-5.381,14.25,0,19.68l4.429,4.439c5.381,5.394,14.22,5.394,19.64,0l71.66-71.83,71.65,71.83c5.381,5.394,14.23,5.394,19.65,0l4.429-4.439c5.381-5.394,5.381-14.25,0-19.68l-71.19-71.83,71.66-71.83c5.381-4.914,5.381-13.78,0-19.21l-4.429-4.423c-5.381-5.394-14.22-5.394-19.64,0l-72.13,71.34-71.17-71.83c-2.691-2.697-6.257-4.044-9.819-4.044z'/%3E%3C/svg%3E");
+            color: @ffe-red;
+            content: '\2717';
+        }
+    }
+
     &--bg-sand {
         > li {
```

Comment:
Maybe this should be changed from styling the raw element and have a `.ffe-check-list__item` class instead? Would also mean not having to have a modifier for a non-existent class below, pluss we moved away from styling on raw elements somewhere around `v4` or something like that?